### ISSUE TITLE
Wrong indent error

### ISSUE THREAD

**Issue by StillFantastic** [protofire/solhint#168]
Here is part of my solidity code:
```
  function fibonacci(uint256 n) internal returns (uint256) {
    if (n == 0) return start;
    else if (n == 1) return start + 1;
    else if (n == 2) return start + 2;
    else return fibonacci(n - 1) + fibonacci(n - 3);
  }
```

After running `solhint **/*.sol`, I got this error message
>   19:10  error  Expected indentation of 11 spaces but found 9    indent

which is at line `else return fibonacci(n - 1) + fibonacci(n - 3);`

Here is my `.solhint.json` file:
```
{
  "extends": "solhint:default",
  "rules": { 
    "indent": ["error", 2],
    "array-declaration-spaces": "error",
    "bracket-align": "error",
    "expression-indent": "error",
    "no-mix-tabs-and-spaces": "error",
    "no-spaces-before-semicolon": "error",
    "space-after-comma": "error",
    "statement-indent": "error",
    "const-name-snakecase": "error",
    "contract-name-camelcase": "error",
    "event-name-camelcase": "error",
    "func-name-mixedcase": "error",
    "func-param-name-mixedcase": "error",
    "modifier-name-mixedcase": "error",
    "var-name-mixedcase": "error",
    "imports-on-top": "error"
  }
}
```

**Comment by fvictorio** [comment]
Hey @StillFantastic, we removed styling rules in 3.0 to make maintenance easier, and we recommend using [prettier-solidity](https://github.com/prettier-solidity/prettier-plugin-solidity), so I will close this issue. Sorry for the late response.

**Comment by willKim19** [comment]
@fvictorio Hi, I'm still running into this issue. I have solhint 3.2.2 installed in my project, but I suspect that maybe the solhint plugin is maybe using the wrong solhint? Have you experienced anything like this?

**Comment by peterblockman** [comment]
@0xGrapefruit did you solve it? I am having the same issue in the latest version of solhint.

**Comment by 0xGrapefruit** [comment]
@peterblockman no I think I gave up

**Comment by fvictorio** [comment]
@peterblockman are you using the `hardhat-solhint` plugin?
